### PR TITLE
Created Footer

### LINKED ISSUE #3: Create the footer
Create a business footer

### PR DESCRIPTION
**Pull request by tharuka01**
## Purpose

The purpose of this PR is to fix #3

### Preview Link

https://pr-{PR_NUMBER}-dine-ease.surge.sh/

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -61,18 +78,163 @@
...
+      </div>
+    </section>
+    <!-- Section: Links  -->
+
+    <!-- Copyright -->
+    <div
+         class="text-center p-3"
+         style="background-color: rgba(0, 0, 0, 0.2)"
+         >
+      © 2020 Copyright:
```
Comment (YoshithaRathnayake) [inline index.html:218]:
```suggestion
      © 2023 DineEase | Browse Restaurants
```

**Review by YoshithaRathnayake — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (YoshithaRathnayake) [inline index.html]:
```suggestion
      © 2023
```

File: index.html [hunk 2]
```diff
@@ -61,18 +78,163 @@
...
+      <a class="text-white" href="https://dineease.org/"
+         >DineEase.org</a
```
Comment (YoshithaRathnayake) [inline index.html]:
```suggestion
         > DineEase | Browse Restaurants</a
```

**Review by YoshithaRathnayake — CHANGES_REQUESTED** [review]

**Review by YoshithaRathnayake — CHANGES_REQUESTED** [review]
image
 
Remove the underline of these links @tharuka01

**Review by YoshithaRathnayake — CHANGES_REQUESTED** [review]
image

Can you see the underscore after every icon without github one?
Try to remove that pls

**Comment by tharuka01** [comment]
> image
> 
> Remove the underline of these links @tharuka01

Fixed

File: assets/css/styles.css [hunk 3]
```diff
@@ -19,6 +19,9 @@
     }
 }
 
+a {
```
Comment (YoshithaRathnayake) [inline assets/css/styles.css]: Just remove this bcs I also add the same

```suggestion
```

File: assets/css/styles.css [hunk 4]
```diff
@@ -19,6 +19,9 @@
     }
 }
 
+a {
+    text-decoration: none;
```
Comment (YoshithaRathnayake) [inline assets/css/styles.css]:
```suggestion
```

File: assets/css/styles.css [hunk 5]
```diff
@@ -19,6 +19,9 @@
     }
 }
 
+a {
+    text-decoration: none;
+}
```
Comment (YoshithaRathnayake) [inline assets/css/styles.css]:
```suggestion
```

**Review by YoshithaRathnayake — CHANGES_REQUESTED** [review]

**Pull request closed without merging**